Diagnostics/HealthChecker/DataCollection/ExchangeInformation/IISInformation/Get-IISWebSite.ps1

```diff
@@ -25,9 +25,18 @@ function Get-IISWebSite {
         $configurationFilePath = (Get-WebConfigFile "IIS:\Sites\$($site.Name)").FullName
         $webConfigExists = Test-Path $configurationFilePath
         $webConfigContent = $null
+        $validWebConfig = $false
 
         if ($webConfigExists) {
             $webConfigContent = Get-Content $configurationFilePath -Raw
+
+            try {
+                [xml]$webConfigContent | Out-Null
+                $validWebConfig = $true
+            } catch {
+                # Inside of Invoke-Command, can't use Invoke-CatchActions
+                Write-Verbose "Failed to convert IIS web config '$configurationFilePath' to xml. Exception: $($_.Exception)"
```

Comment:
```suggestion
                Write-Verbose "Failed to convert IIS web config '$($configurationFilePath)' to xml. Exception: $($_.Exception)"
```

Think we need to change this as suggested.

Reply:
`$configurationFilePath` is a string so you shouldn't need to do that. Only if it is an object where you are doing something like `$configurationFilePath.Attribute` then you would need to do that.

Reply:
You're right:
<img width="610" alt="image" src="https://user-images.githubusercontent.com/40993616/219719562-752b846d-98a3-4a47-a3a4-0394eaf42074.png">